### PR TITLE
Better log texts concerning UsabILIty Hub integration

### PR DESCRIPTION
**Pull request by signedav**

### CODE REVIEW

**Comment by signedav** [comment]
@m-kuhn Before I make the translations I wanted to make proper logs. You might have a quick look.

File: QgisModelBaker/gui/generate_project.py [hunk 1]
```diff
@@ -431,7 +431,7 @@ def accepted(self, edited_command=None):
                                     custom_layer_order_structure = layertree_data['layer-order']
                             except yaml.YAMLError as exc:
                                 self.print_info(
-                                    self.tr('Unable to parse layertree file: {}..').format(exc), COLOR_TOPPING)
+                                    self.tr('Unable to parse layertree structure: {}…').format(exc), COLOR_TOPPING)
```
Comment (signedav) [inline QgisModelBaker/gui/generate_project.py:434]:
```suggestion
                                    self.tr('Unable to parse layertree structure: {}').format(exc), COLOR_TOPPING)
```

File: QgisModelBaker/gui/generate_project.py [hunk 2]
```diff
@@ -474,7 +474,7 @@ def accepted(self, edited_command=None):
                                                    qml_section[layer_qml], 1)
                     if matches:
                         style_file_path = matches[0].data(int(IliToppingFileItemModel.Roles.LOCALFILEPATH))
-                        self.print_info(self.tr('Applying topping on layer {}:{}').format(layer.alias, style_file_path),
+                        self.print_info(self.tr('Applying QML topping on layer {}:{}').format(layer.alias, style_file_path),
```
Comment (signedav) [inline QgisModelBaker/gui/generate_project.py:477]:
```suggestion
                        self.print_info(self.tr('Apply QML topping on layer {}:{}…').format(layer.alias, style_file_path),
```

File: QgisModelBaker/gui/generate_project.py [hunk 3]
```diff
@@ -989,7 +989,7 @@ def get_topping_file_list(self, id_list):
             if matches:
                 file_path = matches[0].data(int(topping_file_model.Roles.LOCALFILEPATH))
                 self.print_info(
-                    self.tr('- - Got file {}..').format(file_path), COLOR_TOPPING)
+                    self.tr('- - Got file {}…').format(file_path), COLOR_TOPPING)
```
Comment (signedav) [inline QgisModelBaker/gui/generate_project.py:992]:
```suggestion
                    self.tr('- - Got file {}').format(file_path), COLOR_TOPPING)
```

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**